vortex-btrblocks/src/integer.rs

```diff
@@ -241,19 +241,18 @@ impl Scheme for FORScheme {
 
         // Difference between max and min
         let full_width: u32 = stats.src.ptype().bit_width().try_into().vortex_unwrap();
-        let padding = 64 - full_width;
-        let bw = full_width + padding - stats.typed.max_minus_min().leading_zeros();
+        let bw = match stats.typed.max_minus_min().checked_ilog2() {
```

Comment:
does rust turn ilog2 into a popcnt?

Reply:
oh huh yea it does https://play.rust-lang.org/?version=stable&mode=release&edition=2024&gist=a2202e4da2892951bd511b74507d1f8d

nice

Reply:
Do you think this makes a difference to perf?

Reply:
no, its just less code and more resilient to changes